### PR TITLE
improve information when intl api is not supported by browser

### PR DESCRIPTION
**Pull request by Jenselme**
- Explain how to use with webpack
- Add logging if Intl API is not supported by browser

### CODE REVIEW

File: src/aurelia-i18n.js [hunk 1]
```diff
@@ -51,16 +51,17 @@ function configure(frameworkConfig, cb): Promise<void> {
 
   // check whether Intl is available, otherwise load the polyfill
   if (window.Intl === undefined) {
+    console.info('Intl API is not available. Trying to load the polyfill.');  // eslint-disable-line no-console
```
Comment (EisenbergEffect) [inline src/aurelia-i18n.js]: We don't want to use console.log here. Instead, we would want to use the logger api. Can you update this PR with that change?

File: src/aurelia-i18n.js [hunk 2]
```diff
@@ -51,16 +51,17 @@ function configure(frameworkConfig, cb): Promise<void> {
...
       return loader.normalize('intl', i18nName).then((intlName) => {
         return loader.loadModule(intlName).then((poly) => {
           window.Intl = poly;
           return registerI18N(frameworkConfig, cb);
-        });
-      });
-    });
+        }, () => console.warn('Failed to load the Intl polyfill.'));  // eslint-disable-line no-console
+      }, () => console.warn('Failed to normalize intl module while loading the Intl polyfill.'));  // eslint-disable-line no-console
+    }, () => console.warn('Failed to normalize aureli-i18n while loading the Intl polyfill.'));  // eslint-disable-line no-console
```
Comment (EisenbergEffect) [inline src/aurelia-i18n.js]: We've got some duplicated error message strings. Can you factor those out into a const and reference that? Also, please switch this to using the logger. Thanks!

**Comment by Jenselme** [comment]
@EisenbergEffect just updated the PR to take your comments into account.

**Comment by zewa666** [comment]
@Jenselme awesome, thanks for the contribution

**Pull request merged**